src/CoreWlan/CWInterface.cs

```diff
@@ -10,10 +10,10 @@
 
 namespace CoreWlan {
 	public unsafe partial class CWInterface {
-		public CWChannel []? SupportedWlanChannels  {
+		public CWChannel []? SupportedWlanChannels {
 			get {
 				NSSet? channels = _SupportedWlanChannels;
-				return channels?.ToArray<CWChannel> ();;
+				return channels?.ToArray<CWChannel> (); ;
```

Comment:
```suggestion
				return channels?.ToArray<CWChannel> ();
```
@rolfbjarne  better to remove the extra ';', right?